File: Ejercicios/MovieRental/Before/MovieRental.ClassLibrary/Rental.cs
namespace MovieRental.ClassLibrary
{
    public class Rental
    {
        public int DaysRented { get; private set; }

        public Movie Movie { get; private set; }

        public Rental(Movie movie, int daysRented)
        {
            this.Movie = movie;
            this.DaysRented = daysRented;
        }
    }
}

File: Ejercicios/Java/MovieRental/Before/src/main/movierental/Rental.java
package movierental;

public class Rental
{
	private int daysRented;
	private Movie movie;

    public Rental(Movie movie, int daysRented)
    {
        this.setMovie(movie);
        this.setDaysRented(daysRented);
    }

	public int getDaysRented() {
		return daysRented;
	}

	public void setDaysRented(int daysRented) {
		this.daysRented = daysRented;
	}

	public Movie getMovie() {
		return movie;
	}

	public void setMovie(Movie movie) {
		this.movie = movie;
	}
}